css-values-5/Overview.bs

```diff
@@ -945,8 +945,47 @@ Generating/Caching Random Values: the <<random-caching-options>> value</h4>
 		</pre>
 	</div>
 
+<!--
+████████ ██       ████████ ██     ██ ████████ ██    ██ ████████        ██████   ███████  ██     ██ ██    ██ ████████ ████ ██    ██  ██████  
+██       ██       ██       ███   ███ ██       ███   ██    ██          ██    ██ ██     ██ ██     ██ ███   ██    ██     ██  ███   ██ ██    ██ 
+██       ██       ██       ████ ████ ██       ████  ██    ██          ██       ██     ██ ██     ██ ████  ██    ██     ██  ████  ██ ██       
+██████   ██       ██████   ██ ███ ██ ██████   ██ ██ ██    ██          ██       ██     ██ ██     ██ ██ ██ ██    ██     ██  ██ ██ ██ ██   ████
+██       ██       ██       ██     ██ ██       ██  ████    ██          ██       ██     ██ ██     ██ ██  ████    ██     ██  ██  ████ ██    ██ 
+██       ██       ██       ██     ██ ██       ██   ███    ██          ██    ██ ██     ██ ██     ██ ██   ███    ██     ██  ██   ███ ██    ██ 
+████████ ████████ ████████ ██     ██ ████████ ██    ██    ██           ██████   ███████   ███████  ██    ██    ██    ████ ██    ██  ██████  
+-->
+
+<h2 id=element-counting>
+Counting Elements</h2>
+
+	Authors often want to apply element specific styles
+	based on the position of the element within the tree.
+
+	The ''sibling-index()'' and ''sibling-count()'' functions
+	(the <dfn export for=CSS>element counting functions</dfn>)
+	allow authors to take these factors into account in ''calc()'' expressions.
+
+	With these, authors can create things like staggered animation delays
+	or easily position elements on a circle.
+
+<h3 id=random>
+Getting the position of an element within its siblings: the ''sibling-index()'' function</h3>
+
+	The <dfn>sibling-index()</dfn> function resolves to the position
+	of an element within its group of <a>inclusive siblings</a>.
```

Comment:
Might want to clarify that it's the inclusive _flat tree_ siblings, which is what we want (I assume).

Reply:
Clarified in 30de15b, but not sure if my wording is 100% correct.

Reply:
We don't use flat tree order for :nth-child(). I think we want :nth-*() and sibling-index() to be consistent?

Reply:
That's true.